### PR TITLE
Joy issue squashing

### PR DESCRIPTION
**Pull request by pm0kjp**
This PR fixes three issues: #868, #889, #904

### CODE REVIEW

File: sql_basics/sql_basics.md [hunk 1]
```diff
@@ -954,5 +955,9 @@ You also got to practice hands on, which probably meant you got to see some erro
 
 * Tutorials Point has some helpful documentation you may want to check out [about the basic types of operators available for use in a SQL query](https://www.tutorialspoint.com/sql/sql-operators.htm).
 
+* Enjoy learning with real-life, consequential examples?  You might enjoy [Select Star SQL](https://selectstarsql.com/), a free interactive book that allows you to run queries against real-world death row data.
+
+* Prefer a game?  The fun and engaging [SQL Murder Mystery](https://mystery.knightlab.com/) might help you hone your skills.
```
Comment (rosemm) [inline sql_basics/sql_basics.md:960]:
```suggestion
* Prefer a game?  The fun and engaging [SQL Murder Mystery](https://mystery.knightlab.com/) or [Lost at SQL](https://lost-at-sql.therobinlord.com/) might help you hone your skills.
```

File: sql_intermediate/sql_intermediate.md [hunk 2]
```diff
@@ -740,6 +740,11 @@ FROM generations;
 
 * Tutorials Point has some helpful documentation you may want to check out [about the basic types of operators available for use in a SQL query](https://www.tutorialspoint.com/sql/sql-operators.htm).
 
+* Enjoy learning with real-life, consequential examples?  You might enjoy [Select Star SQL](https://selectstarsql.com/), a free interactive book that allows you to run queries against real-world death row data.
+
+* Prefer a game?  The fun and engaging [SQL Murder Mystery](https://mystery.knightlab.com/) might help you hone your skills.
```
Comment (rosemm) [inline sql_intermediate/sql_intermediate.md:745]:
```suggestion
* Prefer a game?  The fun and engaging [SQL Murder Mystery](https://mystery.knightlab.com/) or [Lost at SQL](https://lost-at-sql.therobinlord.com/) might help you hone your skills.
```

**Review by rosemm — APPROVED** [review]
LGTM! I added links to lost at SQL in addition to the SQL murder mystery game :)

File: sql_basics/sql_basics.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 module_id: sql_basics
 author: Peter Camacho; Joy Payton
 email: camachop@chop.edu; paytonk@chop.edu
-version: 1.2.2
-current_version_description: Improve large table display with collapsible sections
+version: 1.3.0
+current_version_description: Add two new additional resources
```
Comment (pm0kjp) [inline sql_basics/sql_basics.md:6]:
```suggestion
current_version_description: Add three new additional resources
```

File: sql_intermediate/sql_intermediate.md [hunk 4]
```diff
@@ -2,8 +2,8 @@
 module_id: sql_intermediate
 author:   Peter Camacho; Joy Payton
 email:    camachop@chop.edu
-version: 1.3.0
-current_version_description: Correct data types (numeric), update quiz question, replace Regular Expression section with link to dedicated Regex modules.
+version: 1.4.0
+current_version_description: Add two new additional resources
```
Comment (pm0kjp) [inline sql_intermediate/sql_intermediate.md:6]:
```suggestion
current_version_description: Add three new additional resources
```

File: sql_joins/sql_joins.md [hunk 5]
```diff
@@ -2,8 +2,8 @@
 module_id: sql_joins
 author:   Joy Payton
 email:    paytonk@chop.edu
-version: 1.1.7
-current_version_description: Typo fix; update metadata
+version: 1.2.0
+current_version_description: Add two new additional resources
```
Comment (pm0kjp) [inline sql_joins/sql_joins.md:6]:
```suggestion
current_version_description: Add three new additional resources
```

**Pull request merged**